### PR TITLE
Keane

### PR DESCRIPTION
**Pull request by Keane-Tan**

### CODE REVIEW

**Comment by kpedro88** [comment]
It might not be a good idea to add a 15MB text file to the repository.

File: test.py [hunk 1]
```diff
@@ -22,24 +62,71 @@ def process(self, df):
...
         dataset = df['dataset']
+
+        jets = JaggedCandidateArray.candidatesfromcounts(
+            df['Jets'].counts,
+            px=df['Jets'].fP.fX.flatten(),
+            py=df['Jets'].fP.fY.flatten(),
+            pz=df['Jets'].fP.fZ.flatten(),
+            energy=df['Jets'].fE.flatten(),
+            axismajor=df['Jets_axismajor'].flatten(),
+            axisminor=df['Jets_axisminor'].flatten(),
```
Comment (kpedro88) [inline test.py:73]: we also have `Jets_ptD` for AK4 substructure

**Comment by Keane-Tan** [comment]
oh, I should have paid more attention! I’ll remove the text files!

Best,
Keane

**Comment by kpedro88** [comment]
To remove them from the repository completely, you need to overwrite your branch so they were never committed in the first place (e.g. via squash).

**Pull request merged**